src/components/License/License.js

```diff
@@ -144,6 +154,7 @@ class License extends React.Component {
 
     if (isLoading) return this.renderLoadingPane();
 
+    const licenseId = get(data, ['license', 'id']);
```

Comment:
Can you double-check that this and [the check on L184](https://github.com/folio-org/ui-licenses/pull/174/files#diff-73f0aa0c9055f01dee098610e03beb12R184) are necessary? For example, by `console.log` out `data.license.id` on this line to see if it's ever undefined ?

One of my goals when refactoring ui-licenses was that we could guarantee that `license` was defined and fetched when we got past the `isLoading` check on the previous line. So I kinda think that it'll always be defined and we can clean this render logic up a bit.

That would let you

Reply:
... That would let me what? I'm curious ;-)
I made all the changes.
Thanks for your comments.

Reply:
I'm not sure! 😆 I think probably something along the lines "...clean up the render logic" and then I decided to just put that into the previous paragraph!